Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -1477,13 +1496,76 @@ def malop_to_incident(malop: str) -> dict:
     if not isinstance(malop, dict):
         raise ValueError("Cybereason raw response is not valid, malop is not dict")
 
+    status = 0
+    malopStatus = ""
+    if malop.get('status', ''):
+        malopStatus = (malop.get('status', 'UNREAD'))
+    elif malop.get('simpleValues', ''):
+        malopStatus = (malop.get('simpleValues', {}).get('managementStatus', {}).get('values', ['UNREAD'])[0])
+    if (malopStatus == "Active") or (malopStatus == "UNREAD"):
```

Comment:
You can change it to this:
```suggestion
    if (malopStatus == "Remediated") or (malopStatus == "TODO"):
        status = 1
    elif (malopStatus == "Closed") or (malopStatus == "RESOLVED"):
        status = 2
    else:
        status = 0
```

Reply:
Done